src/page/blog/draft-page.jsx

```diff
@@ -30,12 +30,16 @@ export default function Drafts() {
 
   return (
     <>
-      <div className="w-full max-auto max-w-2xl h-40 left-60 relative">
+      <div
+        className="w-full max-auto max-w-2xl h-40 left-60 relative"
+        data-test-id="articles-page-draft"
+      >
         {isLoggedIn &&
           data.posts.map((post, idx) => {
             const { title, author, createdAt, id } = post;
             return (
               <Draft
+                data-test-id="draft-page-draft"
```

Comment:
```suggestion
```
After testing this out, it turns out the data-attribute does not get applied to the parent div through this. We can probably just address it through enhancing the sub components later.